articles/200_migration_guide_from_ros1.md

```diff
@@ -152,6 +199,47 @@ For more details please see the article about the [generated C++ interfaces](htt
 NOTE: to be written
 </div>
 
+#### Boost
+
+Much of the functionality previously used in boost has been integrated into C++11.
+As such we would like to take advantage of the new core features and drop the dependency on boost.
```

Comment:
We will likely not be able to drop the dependency. But we will use the C++11 equivalent were available.